src/commands/objdump.rs

```diff
@@ -62,6 +62,10 @@ pub struct ObjdumpCommand {
     /// Whether or not to show information about what instructions can trap.
     #[arg(long)]
     traps: bool,
+
+    /// Whether or not to show information about stack maps.
+    #[arg(long)]
+    stack_maps: bool,
```

Comment:
This one also seems like it should probably be on by default and won't be too noisy or anything like that.

Reply:
Ok I've done some rejiggering of the types and such for clap and I think I got something working, namely `--traps` turns things on, `--traps=false` turns things off, and it's on-by-default.